workbench-core/authorization/config/jest.config.json

```diff
@@ -3,10 +3,10 @@
   "collectCoverage": true,
   "coverageThreshold": {
```

Comment:
you can add `"coveragePathIgnorePatterns": ["<rootDir>/src/index.ts"],` here to ignore code coverage on the index file